tests/integration_tests/test_networking.py

```diff
@@ -189,11 +200,54 @@ def test_schema_warnings(
     expected["network"]["ethernets"]["eth0"]["match"]["macaddress"] = mac_addr
     with session_cloud.launch(launch_kwargs=launch_kwargs) as client:
         result = client.execute("cloud-init status --format=json")
-        assert result.failed
-        assert result.return_code == 2  # Warning
+        if CURRENT_RELEASE.series in ("focal", "jammy", "lunar", "mantic"):
```

Comment:
```suggestion
        if CURRENT_RELEASE < NOBLE:
```
with import from tests.integration_tests.releases